automated install/basic-install.sh

```diff
@@ -701,8 +701,11 @@ update_pacakge_cache() {
 
   echo ":::"
   echo -n "::: Updating local cache of available packages..."
-  ${UPDATE_PKG_CACHE} &> /dev/null
-  echo " done!"
+  if eval ${UPDATE_PKG_CACHE} &> /dev/null; then
```

Comment:
When I can figure out a solution that doesn't include `eval` it's going in. I'm not a fan of the kludge either.